scripts/openroad/or_cts.tcl

```diff
@@ -40,9 +40,10 @@ repair_clock_inverters
 puts "\[INFO\]: Configuring cts characterization..."
 configure_cts_characterization\
     -max_slew $max_slew\
-    -max_cap $max_cap\
-    -sqr_cap $::env(CTS_SQR_CAP)\
-    -sqr_res $::env(CTS_SQR_RES)
+    -max_cap $max_cap
+
+cts::set_cap_per_sqr $::env(CTS_SQR_CAP)
+cts::set_res_per_sqr $::env(CTS_SQR_RES) 
```

Comment:
Do not called functions outside the global namespace.  I don't think these commands are required but if they are you should use configure_cts_characterization.  We do not use them in ORFS.